nginx/nginx.conf.dev.template

```diff
@@ -9,6 +11,10 @@ http {
     include mime.types;
     default_type application/octet-stream;
 
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+    access_log  logs/access.log  main;
```

Comment:
Same here
```
iscscfr-nginx-1          | 2023/04/08 09:56:33 [emerg] 9#9: open() "/etc/nginx/logs/access.log" failed (2: No such file or directory)
```

Reply:
fixed in [7351f76](https://github.com/iScsc/iscsc.fr/pull/86/commits/7351f76b6a6753f740f427b580e6c39d9c320981)

but as stated in the commit body I'll soon open a PR to use these

(same reply)